indexers/contacts.go

```diff
@@ -47,6 +48,9 @@ func (i *ContactIndexer) Index(db *sql.DB, rebuild, cleanup bool) (string, error
 
 	// doesn't exist or we are rebuilding, create it
 	if physicalIndex == "" || rebuild {
+		_ = json.Unmarshal(contactsSettingsFile, &contactsSettings)
```

Comment:
should handle this error

Reply:
ah just noticed this is unmarshaling an embedded file so shouldn't ever error - anyways I'll merge and tweak to use `jsonx.MustUnmarshal`